<?xml version="1.0"?>
<launch>
  <node name="dataset_convertor" pkg="msckf2" type="dataset_convertor" output="screen" args="$(arg bag)">  
    <rosparam file="$(find msckf2)/config/dataset_converter.yaml" command="load" ns="/"/>
  </node>
</launch>
